websites/P/Puray Animes/presence.ts

```diff
@@ -0,0 +1,61 @@
+const presence = new Presence({
+		clientId: "972246349917610054"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", () => {
+	const defaultData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp
+		},
+		{ pathname } = window.location,
+		data: PresenceData = defaultData;
+	if (document.querySelector("input[id^=headlessui]"))
+		data.details = "🔍・Pesquisando Animes";
+	else if (pathname.startsWith("/home")) {
+		data.details = "Início:";
+		data.state = "Visualizando animes.";
```

Comment:
presence data state and details should not have punctuation marks. Also you cannot be "viewing animes" in the home page, if anything you'd be searching or browsing them